bench/compiled_nbt.js

```diff
@@ -3,22 +3,19 @@ const { performance } = require('perf_hooks')
 const assert = require('assert')
 const { ProtoDefCompiler } = require('protodef').Compiler
 const fs = require('fs')
+const nbt = require('../nbt')
 
 const mainType = 'nbt'
 
-fs.readFile('../sample/bigtest.nbt', async (error, buffer) => {
-  if (error) {
-    throw error
-  }
-
-  const proto = new ProtoDef()
-  proto.addTypes(require('../compound'))
-  proto.addTypes(require('../nbt.json'))
+function main (nbTests = 10000) {
+  const buffer = fs.readFileSync(__dirname + '/../sample/bigtest.nbt') // eslint-disable-line n/no-path-concat
+  const validate = false
```

Comment:
This fails with validation on with protodef-validator
```json
{
  "keyword": "enum",
  "dataPath": "",
  "schemaPath": "#/oneOf/0/enum",
  "params": {
    "allowedValues": [
      "native"
    ]
  },
  "message": "should be equal to one of the allowed values",
  "schema": [
    "native"
  ],
  "parentSchema": {
    "enum": [
      "native"
    ]
  },
  "data": [
    "container",
    [
      {
        "name": "type",
        "type": "nbtMapper"
      },
      {
        "name": "name",
        "type": "nbtTagName"
      },
      {
        "name": "value",
        "type": [
          "nbtSwitch",
          {
            "type": "type"
          }
        ]
      }
    ]
  ]
}
```

The tests run ok disabling the validation

Reply:
is some type not added as native? how come it fails validation?

Reply:
Ah yeah looks like missing native was the cause, should be fixed now